docs/account-sharing/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 1
+sidebar_label: Account Sharing
+title: Account Sharing
+description: Learn how to share your Momento account with other users.
+pagination_next: null
+hide_title: true
+keywords:
+  - account sharing
+  - sharing
+  - collaboration
+  - team
+  - organization
+  - resources
+  - members
+  - accounts
+---
+
+
+# What is Account Sharing?
+Momento's Account Sharing feature enables you to collaborate and manage your resources with ease. By sharing your account, you can grant other users access to your topics, caches, and other resources, fostering seamless teamwork and organization.
+
+## Overview
+Account Sharing allows you to efficiently manage your team and resources. You can easily add or remove members from your account, ensuring that the right people have the right access. This feature is designed to optimize workflows and enhance collaboration within your organization.
+## How to Use Account Sharing
+To get started with Account Sharing, follow these steps:
+
+### **Add a Member**
+    - Log in to the [Momento Console](https://console.gomomento.com).
+    - In the upper-right corner, click on your account info dropdown and select **Manage Account**.
+
+        <img src="/img/account-sharing/manage-account.png" width="60%"/>
+    - Click on the **Add Member** button.
+
+        <img src="/img/account-sharing/add-member.png" width="60%"/>
+    - Enter the email address of the user you wish to add. Please ensure that the user has an existing Momento account, meaning they must have signed into the Momento console at least once using that email address.
```

Comment:
what happens if they type an e-mail address that has never logged in to the Momento console?

Reply:
The user will not be identified by the server and we throw an error on console saying "User could not be found."

Reply:
okay. i just tested it and I see what you mean. This PR is good then, but we should create a ticket for the console itself to augment that error message a bit, so that users will know what the remedy is if they get that error.